Autopilot/Src/timeStampingPOGI.cpp

```diff
@@ -1,10 +1,10 @@
 
-#include "timeStampingPOGI.hpp"
+#include "TimeStampingPOGI.hpp"
```

Comment:
should be `timeStampingPOGI.hpp`. I am surprised this built on your local... what OS are you using?

Reply:
hmm, that's weird. I renamed the file to TimeStampingPOGI on my local end, I guess those changes didn't sync through git?

Reply:
Still need to change the file name in this `#include`